ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -63,30 +65,30 @@ to answer all of them.
    to be handed off to the responsible Area Director?
 
 10. Several IETF Areas have assembled [lists of common issues that their
-    reviewers encounter][6]. Do any such issues remain that would merit specific
-    attention from subsequent reviews?
+    reviewers encounter][6]. Are any such issues still present in the document?
```

Comment:
How about:
```suggestion
    reviewers encounter][6]. For which areas have such issues been identified and addressed, and
    for which areas does this still need to happen in subsequent reviews?
```

Reply:
This is better, but it needs some grammar work -- e.g., 'for which... for which'.